SPECS-EXTENDED/libguestfs/libguestfs.spec

```diff
@@ -36,12 +36,14 @@ Distribution:   Mariner
 # Unbreak the linker.
 %undefine _strict_symbol_defs_build
 
-%bcond_with applicances
+%bcond_without applicances
+%bcond_with php
```

Comment:
I think this is not used anywhere in the spec.

Reply:
Forgot to cherry-pick a commit into my staging branch. Re-added the usage of this bcond.